packages/plugin-ext/src/hosted/browser/worker/worker-main.ts

```diff
@@ -49,16 +49,24 @@ const rpc = new RPCProtocolImpl({
         ctx.postMessage(m);
     },
 },
-{
-    reviver: reviver
-});
+    {
+        reviver: reviver
+    });
```

Comment:
We all seem to use different formatters :)

I think you can move the opening bracket from L52 to the end of L51 and the format should be more consistent.